src/main/kotlin/no/nav/dagpenger/regel/api/RegelApi.kt

```diff
@@ -0,0 +1,193 @@
+package no.nav.dagpenger.regel.api
+
+import de.nielsfalk.ktor.swagger.SwaggerSupport
+import de.nielsfalk.ktor.swagger.description
+import de.nielsfalk.ktor.swagger.example
+import de.nielsfalk.ktor.swagger.examples
+import de.nielsfalk.ktor.swagger.get
+import de.nielsfalk.ktor.swagger.ok
+import de.nielsfalk.ktor.swagger.post
+import de.nielsfalk.ktor.swagger.responds
+import de.nielsfalk.ktor.swagger.version.shared.Group
+import de.nielsfalk.ktor.swagger.version.shared.Information
+import de.nielsfalk.ktor.swagger.version.v2.Swagger
+import io.ktor.application.Application
+import io.ktor.application.call
+import io.ktor.application.install
+import io.ktor.features.CallLogging
+import io.ktor.features.ContentNegotiation
+import io.ktor.features.DefaultHeaders
+import io.ktor.gson.gson
+import io.ktor.http.HttpHeaders
+import io.ktor.http.HttpStatusCode
+import io.ktor.locations.Location
+import io.ktor.locations.Locations
+import io.ktor.response.header
+import io.ktor.response.respond
+import io.ktor.routing.Routing
+import io.ktor.routing.routing
+import io.ktor.server.engine.embeddedServer
+import io.ktor.server.netty.Netty
+import mu.KotlinLogging
+
+private val LOGGER = KotlinLogging.logger {}
+
+data class MinsteInntektBeregningsRequest(
+    val aktorId: String,
+    val verneplikt: Boolean,
+    val fangstOgFisk: Boolean
+)
+
+data class GrunnlagBeregningsRequest(
+    val aktorId: String,
+    val verneplikt: Boolean,
+    val fangstOgFisk: Boolean
+)
+
+data class TaskResponse(
+    val regel: Regel,
+    val status: TaskStatus,
+    val expires: String
+)
+
+enum class Regel {
+    MINSTEINNTEKT, GRUNNLAG
+}
+
+class RegelApi {
+    companion object {
+        @JvmStatic
+        fun main(args: Array<String>) {
+            embeddedServer(Netty, port = 8092, module = Application::api).start(wait = true)
```

Comment:
```suggestion
           val app =  embeddedServer(Netty, port = 8092, module = Application::api)
           app.start(wait = false)
           Runtime.getRuntime().addShutdownHook(Thread {
                   app.stop(5, 60, TimeUnit.SECONDS)
           })
            
```

A good idea to add a shutdown hook here -